buffer.py

```diff
@@ -1025,47 +1053,57 @@ def toggle_read_mode(self):
     @interactive
     def scroll_up(self):
         self.update_vertical_offset(min(self.scroll_offset + self.scroll_step, self.max_scroll_offset()))
+        self.clear_synctex_indicator()
```

Comment:
After synctex_pos_y become global coordiante, we just need test in paintEvent, don't need write clear_synctex_indicator.